app/views/reports/regions/_children.html.erb

```diff
@@ -0,0 +1,29 @@
+<ul class="access-item-wrapper">
+  <% @children.each do |parent, children| %>
+  <%# use minus 2 for 'state offset' %>
+  <% depth = Region::REGION_TYPES.index(parent.region_type) - 2 %>
+  <%# add more space if there arent any children, because we wont show the arrow icon  %>
+  <% depth = depth + 1 if children.nil? || children.none? %>
+  <li>
+    <div class="access-item collapsed <%= parent.region_type %>">
+      <% depth.times do %>
+        <span class="spacer"></span>
+      <% end %>
+
+      <% if children && children.any? %>
+      <div class="access-item__dropdown">
+        <i class="fas fa-caret-down light"></i>
+      </div>
+      <% end %>
+
+      <div class="form-check show">
+        <%= link_to_if(true, parent.name, reports_region_path(parent, report_scope: parent.region_type)) %>
+      </div>
+
+      <div class="access-item__padding"></div>
+    </div>
+
+    <%= render "children", children: children if children && children.any? %>
+  </li>
+  <% end %>
+</ul>
```

Comment:
Is this no longer required, since we're using the view component?

Reply:
nope, deleting!